### PR TITLE
Exclusionist Items

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds the Exclusionist robes, armor and flags used in the recent event, and adds an Exclusionist gear crate to the uplink. The armor has been modified to be on-par with normal antagonist heavy armor.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Review by Arrow768 — APPROVED** [review]

File: code/datums/uplink/gear loadout.dm [hunk 1]
```diff
@@ -203,3 +203,14 @@
 	desc = "A crate containing gear for a single individual."
 	path = /obj/structure/closet/crate/secure/gear_loadout/tcaf/single
 	telecrystal_cost = 10
+
+/datum/uplink_item/item/gear_loadout/exclusionists
```
Comment (DreamySkrell) [inline code/datums/uplink/gear loadout.dm:207]: should be `exclusionist` or `exclusionist_group`

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 2]
```diff
@@ -1033,6 +1033,39 @@
 	new /obj/item/gun/energy/blaster/tcaf(src)
 	new /obj/item/storage/belt/military(src)
 
+/obj/structure/closet/crate/secure/gear_loadout/exclusionist/fill()
```
Comment (DreamySkrell) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:1036]: same here

File: code/modules/clothing/factions/trinary.dm [hunk 3]
```diff
@@ -22,3 +22,68 @@
...
+		bullet = ARMOR_BALLISTIC_MAJOR,
+		laser = ARMOR_LASER_RIFLE,
+		energy = ARMOR_ENERGY_SMALL,
+		bomb = ARMOR_BOMB_PADDED,
+	)
+	contained_sprite = TRUE
+	slowdown = 1
+	allowed = list(/obj/item/device/flashlight,/obj/item/tank,/obj/item/device/suit_cooling_unit,/obj/item/gun,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/melee/energy/sword,/obj/item/handcuffs)
+
+/obj/item/clothing/mask/exclusionists
```
Comment (DreamySkrell) [inline code/modules/clothing/factions/trinary.dm:76]: `/exclusionist`

**Review by DreamySkrell — DISMISSED** [review]

File: code/modules/clothing/factions/trinary.dm [hunk 4]
```diff
@@ -22,3 +22,68 @@
...
+	name = "exclusionist robes"
+	desc = "Red and gold robes worn by those who follow the Exclusionist heresy of the Trinary Perfection."
+	icon = 'icons/clothing/suits/exclusionist_robes.dmi'
+	icon_state = "exclusionist_robe_priest"
+	item_state = "exclusionist_robe_priest"
+	hoodtype = /obj/item/clothing/head/winterhood/exclusionist
+	contained_sprite = TRUE
+
+/obj/item/clothing/suit/storage/hooded/exclusionist_robe/warpriest
+	name = "exclusionist warpriest's robe"
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/trinary.dm:37]:
```suggestion
	name = "exclusionist warpriest's robes"
```

File: code/modules/clothing/factions/trinary.dm [hunk 5]
```diff
@@ -22,3 +22,68 @@
...
+	name = "exclusionist warpriest's hood"
+	desc = "A black and gold hood worn by those who follow the Exclusionist heresy of the Trinary Perfection."
+	icon_state = "exclusionist_robe_warpriest_hood"
+	item_state = "exclusionist_robe_warpriest_hood"
+
+/obj/item/clothing/ears/antenna/trinary_halo/exclusionist
+	color = "#1c1c1c"
+
+/obj/item/clothing/suit/armor/exclusionist
+	name = "armored exclusionist robe"
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/trinary.dm:60]:
```suggestion
	name = "armored exclusionist robes"
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**